borrowing/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from borrowing.models import Borrowing
+from borrowing.serializers import (
+    BorrowingSerializer,
+    BorrowingListSerializer,
+    BorrowingDetailSerializer
+)
 
-# Create your views here.
+
+class BorrowViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Borrowing.objects.all()
```

Comment:
remember to add optimization request to db

Reply:
Forget this rule
Resolved